src/main/java/org/geysermc/discordbot/listeners/ErrorAnalyzer.java

```diff
@@ -20,20 +20,12 @@
     private final Map<String, String> configExceptionFixes;
     private final Map<Pattern, String> configExceptionChecks;
 
-    private static Pattern BRANCH_PATTERN = Pattern.compile("Geyser .* \\(git-[0-9a-zA-Z]+-([0-9a-zA-Z]{7})\\)");
+    private static final Pattern BRANCH_PATTERN = Pattern.compile("Geyser .* \\(git-[0-9a-zA-Z]+-([0-9a-zA-Z]{7})\\)");
 
-    public ErrorAnalyzer() {
-        configExceptionFixes = new HashMap<>();
+    public ErrorAnalyzer(Map<String, String> exceptionFixes) {
```

Comment:
Should probably not pass this here and instead just reference it straight from the TagsManager, which means we could possibly change how that works later on and also means we don't have 2 copies of the same data in memory.